Pds4File/pds4file.py

```diff
@@ -32,14 +32,17 @@
 # Configuration
 ################################################################################
 
+#VOLTYPES = ['volumes', 'calibrated', 'diagrams', 'metadata', 'previews',
+#            'documents']
 VOLTYPES = ['volumes', 'calibrated', 'diagrams', 'metadata', 'previews',
-            'documents']
+            'documents', 'bundles']
 VIEWABLE_VOLTYPES = ['previews', 'diagrams']
 
 VIEWABLE_EXTS = set(['jpg', 'png', 'gif', 'tif', 'tiff', 'jpeg', 'jpeg_small'])
 DATAFILE_EXTS = set(['dat', 'img', 'cub', 'qub', 'fit', 'fits'])
 
-VOLSET_REGEX        = re.compile(r'^([A-Z][A-Z0-9x]{1,5}_[0-9x]{3}x)$')
+#VOLSET_REGEX        = re.compile(r'^([A-Z][A-Z0-9x]{1,5}_[0-9x]{3}x)$')
+VOLSET_REGEX        = re.compile(r'(^uranus_occs_earthbased)$') # Hard-code for the moment, but will need to generalise. Should rename variable as BUNDLESET_REGEX?
```

Comment:
- Good idea, let's rename ```VOLSET``` to ```BUNDLESET```
- Use ```re.compile(r'^(uranus_occs_earthbased)$')```, and we can generalize this pattern later when we know more bundle set names.